README.md

```diff
@@ -7,3 +7,68 @@
 [godoc-badge]: https://pkg.go.dev/badge/darvaza.org/penne.svg
 [goreport]: https://goreportcard.com/report/darvaza.org/penne
 [goreport-badge]: https://goreportcard.com/badge/darvaza.org/penne
+
+_Penne_ is a config-driven pipeline oriented DNS resolver that allows complex
+flows simply defined.
+_Penne_ is built using the [darvaza sidecar engine][sidecar] and
+the [darvaza resolver interface][resolver].
+
+[core]: https://pkg.go.dev/darvaza.org/core
+[resolver]: https://pkg.go.dev/darvaza.org/resolver
+[sidecar]: https://pkg.go.dev/darvaza.org/sidecar
+[slog]: https://pkg.go.dev/darvaza.org/slog
+
+## Horizons
+
+A _Horizon_ is a named set of network patterns (aka `CIDR`) that can optionally choose a custom `Resolver`
+and can annotate requests before passing them to the next _Horizon_ on a chain.
+
```

Comment:
I would mention here the `split` word as maybe people are more familiar with the `split horizon` sintagm

Reply:
check now